### ISSUE TITLE
No new planet file for 2 weeks

### ISSUE THREAD

**Issue by JesseCrocker** [openstreetmap/operations#203]
Last week there was no new planet file on https://planet.openstreetmap.org/planet/2018/, and it's now 10 hours after the time weekly planet files usually appear and there isn't one for this week.

The last planet file is planet-180226.osm.bz2 uploaded on 2018-03-01.

Apologies if this issue has already been addressed somewhere else, I looked and was not able to find any discussion of it.

**Comment by Firefishy** [comment]
Been fixed a day or 2 ago. See: https://github.com/openstreetmap/chef/commit/0e95d80e29d91ead2a078ee13682268c28e2ffc6

**Comment by Firefishy** [comment]
Cheers for checking :)

**Comment by Firefishy** [comment]
@JesseCrocker new planet file is out. Grab it while it is hot ;-)

**Comment by zerebubuth** [comment]
Apologies for the delay and missing planet last week. We had to fix a couple of unexpected side-effects of the fix for [CVE-2018-1058](https://wiki.postgresql.org/wiki/A_Guide_to_CVE-2018-1058:_Protect_Your_Search_Path), which included [a new version of the `pg_dump` and `pg_restore` tools](https://www.postgresql.org/docs/9.5/static/release-9-5-12.html), including a new format version for the file. Since the new version isn't backwards-compatible with older tools, we had to make sure all the machines which process the planet file, even ones not running PostgreSQL servers, were updated. And then to take account of the newly explicit schema name in the planet dump tool itself (the bump to `1.1.6` that @Firefishy pointed out).

This week's planet was delayed by the time it took me to fix that and kick off a new run. Since that has now completed, I hope that we'll be back to the usual automatic schedule for planet releases.
